controller_manager/src/controller_manager.cpp

```diff
@@ -621,6 +621,16 @@ controller_interface::return_type ControllerManager::configure_controller(
       "update rate.",
       controller_name.c_str(), controller_update_rate, cm_update_rate);
   }
+  else if (controller_update_rate != 0 && cm_update_rate % controller_update_rate != 0)
+  {
+    const auto act_ctrl_update_rate = cm_update_rate / (cm_update_rate / controller_update_rate);
```

Comment:
why this expression?

```
    const auto act_ctrl_update_rate = 1.0 / controller_update_rate;
```

Reply:
Hello @bmagyar !

Thanks for the review. Regarding your question, I had to do it that way to take into account the integer division. Let me give you an example.

If the controller update rate is 40 Hz and then controller manager update rate is 100 Hz
Then` (cm_update_rate/controller_update_rate)` will be 2, then the output of the whole expression`const auto act_ctrl_update_rate = cm_update_rate / (cm_update_rate / controller_update_rate)`  is 50 Hz. That's the main idea behind it.

Thank you,

Best Regards,
Sai

Reply:
Thanks. Since I had to ask about it I think it may be good to leave a comment with a similar explanation for posterity

Reply:
@bmagyar You have a valid point. It totally make sense to leave a comment. Thanks for bringing it up. 

I'll probably update it on Sunday and will update you. 
Thank you

Reply:
Done. I've added the comment.

Reply:
Thx